c/src/ml-api-service-agent-client.c

```diff
@@ -176,7 +183,7 @@ _build_ml_info_from_json_cstr (const gchar * jcstring, void **handle)
   } else {
     if (_info_list)
       ml_information_list_destroy (_info_list);
-    if (_info)
+    else if (_info)
       ml_information_destroy (_info);
```

Comment:
in case of line 174, ```_info``` and ```_info_list``` need to free, right?

Reply:
In the changes, _info is added into _info_list. So If given json is array, destroying _info_list internally releases _info.

Reply:
Thank very much :pray: